GCRCatalogs/register.py

```diff
@@ -558,20 +584,29 @@ def get_root_dir():
     return _config_register.root_dir
 
 
-def set_root_dir(path):
+def set_root_dir(path, write_to_config=False):
     """
     Sets runtime root_dir to *path*.
     """
     _config_register.root_dir = path
 
+    if write_to_config:
+        _config_register.persist_root_dir()
+
 
-def set_root_dir_by_site(site):
+def set_root_dir_by_site(site, write_to_config=False):
```

Comment:
`write_to_config` not used
```suggestion
def set_root_dir_by_site(site):
```